Dockerfile

```diff
@@ -1,22 +1,33 @@
 FROM debian:jessie
```

Comment:
Could we try something like `debian:9.3-slim` while at it?

Reply:
Right, forgot about that. Will look into it.